packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -68,8 +68,24 @@ const styles = (theme: ThemeType): JssStyles => ({
       marginTop: 15
     }
   },
+  messageBtnDesktop: {
+    display: 'block',
+    [theme.breakpoints.down('xs')]: {
+      display: 'none'
+    }
+  },
+  messageBtnMobile: {
+    display: 'none',
+    [theme.breakpoints.down('xs')]: {
+      display: 'block'
+    }
+  },
   messageBtn: {
-    boxShadow: 'none'
+    boxShadow: 'none',
+    marginLeft: 20,
+    [theme.breakpoints.down('xs')]: {
+      margin: '5px 0 10px'
+    }
```

Comment:
Added some more spacing around this "Message" button, and moved it down on mobile since it was a bit cramped next to the username.